cmake/RunCoverage.cmake.in

```diff
@@ -0,0 +1,15 @@
+if($<BOOL:$<CONFIG:Debug>>)
```

Comment:
Please add copyright header & an explanation of the purpose of this file.

Reply:
Sure thing. Good points.